pkg/segment/traits.go

```diff
@@ -0,0 +1,11 @@
+package segment
+
+import (
+	"gopkg.in/segmentio/analytics-go.v3"
+	"runtime"
+)
+
+func traits() analytics.Traits {
```

Comment:
What's the purpose of this function? Are we creating a separate file under `pkg/segment` for just one function? Or do we plan to add more to it going further?

Reply:
Moved it to `segment.go::addConfigTraits`.